ssl/t1_trce.c

```diff
@@ -729,7 +729,7 @@ static int ssl_print_extension(BIO *bio, int indent, int server,
         while (xlen > 0) {
             size_t plen = *ext++;
 
-            if (plen > xlen + 1)
+            if (plen + 1 > xlen)
```

Comment:
Side note: This stuff really needs to be rewritten to use the PACKET stuff. This is a typical example of the kind of issue we used to have before we switched to PACKET everywhere else.

Reply:
> This stuff really needs to be rewritten to use the PACKET stuff

I agree :-)